src/System.Net.HttpListener/src/System/Net/HttpListenerResponse.cs

```diff
@@ -8,6 +8,19 @@ namespace System.Net
 {
     public sealed unsafe partial class HttpListenerResponse : IDisposable
     {
+        private enum ResponseState
+        {
+            Created,
+            ComputedHeaders,
+            SentHeaders,
+            Closed,
+        }
+
+        private ResponseState _responseState = ResponseState.Created;
```

Comment:
Nit: please move the properties down to be with the other props, and the field down to be with the other fields.